src/index.js

```diff
@@ -143,16 +143,24 @@ export const getTargets = (targets = {}) => {
   return targetOps;
 };
 
+export const validateBoolOption = (name, value, defaultValue) => {
+  if (typeof value === "undefined") {
```

Comment:
in this particular case it's part of the plugin so we would just need to move the dep to deps instead of devDeps